lib/variable.js

```diff
@@ -12,15 +12,28 @@ const base64Regex = /^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-
  * @param  {Object} extraAccessors Extra accessors to install.
  * @return {Object}
  */
-module.exports = function getVariableAccessors (container, varName, defValue, extraAccessors) {
+module.exports = function getVariableAccessors (container, varName, extraAccessors) {
   let isBase64 = false
+  let isRequired = false
+  let defValue
+  let example
 
   /**
    * Throw an error with a consistent type/format.
    * @param {String} value
    */
   function raiseError (value, msg) {
-    throw new EnvVarError(`"${varName}" ${msg}, but was "${value}"`)
+    let errMsg = `"${varName}" ${msg}`
+
+    if (value) {
```

Comment:
Probably want to do a check specifically for undefined here, in case it’s a Boolean or zero or other falsey  value. 
EDIT: or are strings the only possible type here?

Reply:
Yep, should only be strings or undefined that get here.